src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -144,4 +145,23 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
         const version: SemVer | null = coerce(this.server.version);
         return !!version && gte(version, '11.0.0');
     }
+
+    public validateIdentifier(identifier: string, identifierLabel: string): string | undefined {
```

Comment:
Why do you need a separate identifierLabel?  Is this not just used to validate names?  Or are there plans to expand its usage?  Would these rules apply to all properties?

Reply:
Those naming rules apply to any postgres identifier, so tables, funcs, stored procs, etc. I think it would be safe to drop `identifierLabel` and just use "Name"